### DISCUSSION TITLE
Wish for adding AZV Landkreis Hof

### DISCUSSION THREAD

**Discussion by JayDi999** [mampfes/hacs_waste_collection_schedule#1820] [Ideas]
Hello I wish you could add the AZV Landkreis Hof 

https://www.azv-hof.de/privat/abfuhrtermine/abfuhrkalender-landkreis-hof.html

Thank you !

**Comment by 5ila5** [comment]
AZV Hof is supported by the [ICS source ](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/ics.md) I added dedicated documentation in #1824 you can find it [here](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/ics/azv_hof_de.md)
